src/JSInterop/Microsoft.JSInterop/src/JSRuntime.cs

```diff
@@ -121,6 +121,20 @@ public ValueTask<TValue> InvokeAsync<TValue>(string identifier, CancellationToke
             }
         }
 
+        /// <summary>
+        /// Invokes the specified JavaScript function asynchronously.
+        /// </summary>
+        /// <param name="identifier">An identifier for the function to invoke. For example, the value <c>"someScope.someFunction"</c> will invoke the function <c>window.someScope.someFunction</c>.</param>
+        /// <param name="cancellationToken">
+        /// A cancellation token to signal the cancellation of the operation. Specifying this parameter will override any default cancellations such as due to timeouts
+        /// (<see cref="JSRuntime.DefaultAsyncTimeout"/>) from being applied.
+        /// </param>
+        /// <param name="args">JSON-serializable arguments.</param>
+        public async ValueTask InvokeVoidAsync(string identifier, CancellationToken cancellationToken, object[] args)
+        {
+            await this.InvokeAsync<VoidReturn>(identifier, cancellationToken, args);
```

Comment:
Out of interest, is it possible to use `System.Void` instead of `VoidReturn`?

Reply:
It doesn't appear that we can use `System.Void` directly:
![image](https://user-images.githubusercontent.com/11198735/84661947-5296b980-aed8-11ea-8b15-abd9ea7e216d.png)

Reply:
We could look at using reflection to invoke the method if you like... something like this:
```
public async ValueTask InvokeVoidAsync(string identifier, CancellationToken cancellationToken, object[] args)
{
    var voidType = typeof(void);
    var invokeAsyncMethod = this.GetType().GetMethod(nameof(InvokeAsync), new Type[] { typeof(string), typeof(CancellationToken), typeof(object[]) }).MakeGenericMethod(voidType);
    var task = (Task)invokeAsyncMethod.Invoke(this, new object[] { identifier, cancellationToken, args });
    await task;
}
```
This compiles but I haven't actually tested the code to see if it works.  If you think it's worth pursuing I can look at getting my environment setup to test and/or create a unit test if you can point me in the right direction as to where those are.  I've just been opening up the project directly since the `JSInterop.slnf` file doesn't work and I'm not actually sure what solution uses this project.

Reply:
I'd rather avoid reflection here if we can. Your previous solution was fine really!